plugins/generator-1.21.1/neoforge-1.21.1/utils/triggers.java.ftl

```diff
@@ -15,6 +15,10 @@
 				"world": "entity.level()",
 				"itemstack": "itemstack"
 			}, false/>));
+		<#elseif translationKeyHeader != "">

```

Comment:
A more FTL-ic check would be ?has_content

Reply:
:heavy_check_mark: Fixed